### PR TITLE
/app stylesheet

### PR DESCRIPTION
**Pull request by jakecatron**
## Problem
*Document current behavior and why it's undesirable here.*

## Solution
*Document how the behavior was fixed or enhanced here.*

### Corresponding Branch
*Add link to crdschurch/repo-name-here#issue-number (if applicable). This helps the code reviewer know that corresponding work exists and where to find it.*

## Testing
*Include information on how to test your work here (if applicable).*

### CODE REVIEW

File: _assets/stylesheets/pages/_app.scss [hunk 1]
```diff
@@ -0,0 +1,220 @@
+
+
+/* iPhone X & Pixel 2 XL */
+@media (max-height: 896px) and (max-width: 414px) {
+  .jumbo-viewport {
+    min-height: 44vh;
```
Comment (lukemoderwell) [inline _assets/stylesheets/pages/_app.scss]: In styles there is the idea of a "magic number". It's basically when someone hardcodes a value to something that looks good. Old article here: https://css-tricks.com/magic-numbers-in-css/ on why they aren't a good thing in css.

File: _assets/stylesheets/pages/_app.scss [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+    margin-top: -34vh;
+  }
+  .jumbotron .font-family-base-light {
+    font-size: 4rem !important;
+  }
+    
+  .font-family-condensed {
+    font-size: 1rem !important;
+  }
+  .logo-margin{
```
Comment (lukemoderwell) [inline _assets/stylesheets/pages/_app.scss]: I probably would have just put a `margin` on the `.logo` class.

File: _assets/stylesheets/pages/_app.scss [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+    }
+  }
+
+  @media (max-width: 767px) {
+    .hide-buttons {
+      visibility: hidden;
+    }
+  }
+/* Jumbotron Styling */
+img.jumbotron {
```
Comment (lukemoderwell) [inline _assets/stylesheets/pages/_app.scss]: Why did you add the `img` tags here? Anything wrong with dropping them?

File: _assets/stylesheets/pages/_app.scss [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+  margin-bottom: 30px;
+}
+
+p a:hover {
+  color: white;
+  text-decoration: none !important;
+  opacity: 0.6;
+}
+
+.crds-shared-header p a {
```
Comment (lukemoderwell) [inline _assets/stylesheets/pages/_app.scss]: I would avoid this type of css. This isn't scoped just to this page and you might high other `a` tags in places you don't want.

File: newApp.html [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+      </a>
+      <a href="https://play.google.com/store/apps/details?id=net.crossroads.crossroads&amp;hl=en">
+        <img src="https://crds-media.imgix.net/7dFtrIsd22DHPYkudlXC4C/bac32e1de443d0765efcbda9f1f3e051/google-button.svg" alt="Google Play" class="app-button" style="margin-right: 30px;" title="Google Play" />
+      </a>
+    </div>
+  </div>
+  <!-- Crossroads Anywhere & Tagline-->
+  <div class="container title-top">
+    <div class="row">
+    <p class="font-family-condensed text-uppercase">Crossroads Anywhere App</p>
```
Comment (lukemoderwell) [inline newApp.html]:
```suggestion
    <p class="font-family-condensed text-uppercase font-size-small">Crossroads Anywhere App</p>
```

This text gets large on mobile.

**Comment by lukemoderwell** [comment]
@jakecatron looks good - only thing I would definitely change is the shared header styles. Let me know if that doesn't make sense.

**Comment by jakecatron** [comment]
> I'd fix Luke's comments and then you and I can pair on the HTML

Yea, I went ahead and removed the shared header styles. We're going to go ahead and get it out in prod so it's not held up any more and then we can make tweaks later. That'd be great to pair with you on the HTML, I know there's a lot of room for improvement.
